File: functions/python-function/runLocal.py
import json

from app import lambda_handler

def run_local():
    try:

        result = lambda_handler({}, {})
        print(json.dumps(result, indent=2))
    except Exception as e:
        import traceback
        traceback.print_exc()

# Execute the local invocation
if __name__ == "__main__":
    run_local()

File: functions/convertSessionDataToJSON/runLocal.js
import { lambdaHandler } from './app.js';
import event from './event.json' with { type: "json" };

async function runLocal() {
  try {
    const response = await lambdaHandler(event);
    console.log(response);
  } catch (error) {
    console.log(error);
  }
}

runLocal();
